TDA/Software-Sec-Final-proj
# Rajat

**Summary:**
Users, user_roles were kept same. New tables : user_requests, transactions and accounts
Modified functionalities according to DB: Credit, Debit, User transfer, Merchant transfer, Edit info, View statement.
**Description:** Every transaction creates an entry in the appropriate table. Accounts table is unused as of now. For now userID is being used as accountID, but later we need to separately register accounts after users signup is done. That accountID will then be used in transactions. Edit Info uses user_requests table which will be used as staged area until approved by admin/manager.
**How to test:**
Create new DB schema using SQL file present with project. Change password to '1234' for database if not already so. Login using credentials "kenilabc", "Bhatt1!". Go to /account relative address. Check each functionality. Check your DB to see that tables are being populated.
**Missing functionality**: Input validation [Bhardwaj is working], database entry validation, calculating balance, minor UI changes.
